_includes/homePage.html

```diff
@@ -89,13 +99,13 @@
 
 <section class="hero"   style="padding-bottom: 1rem">
 	<div class="text-container">
-		<h1 style="margin: 3rem 0rem 3rem 0rem;">the <span class="special-text" style="animation-delay: .5s;"> extensible</span> Kubernetes manager</h1>
+		<h1 style="margin: 3rem 0rem 3rem 0rem;">The <span class="special-text" style="animation-delay: .5s;"> Extensible</span> Kubernetes manager</h1>
```

Comment:
Thanks for considering this change, @Cvr421, but this phrasing is by design and a change is not desired. This is true for all the updates in verbiage.

Reply:
ok got it. But I thought that it feels something incomplete when visiting the page. that's why I make this change. But thank you . it's a learning for me.